src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -132,6 +178,56 @@ public void BasicTest_GetIPInterfaceStatistics_Success()
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Linux)]
+        public void BasicTest_GetIPInterfaceStatistics_Success_Linux()
+        {
+            // This API is not actually IPv4 specific.
```

Comment:
Not sure what that comment is referring to either... It's in the Windows test case so I copied it over.